dragonchain/webserver/routes/interchain.py

```diff
@@ -122,6 +142,19 @@ def update_ethereum_interchain_v1(name: str) -> Tuple[str, int, Dict[str, str]]:
     return helpers.flask_http_response(200, interchain.update_ethereum_interchain_v1(name, data))
 
 
+@request_authorizer.Authenticated()
+def update_binance_interchain_v1(name: str) -> Tuple[str, int, Dict[str, str]]:
+    if not flask.request.is_json:
+        raise exceptions.BadRequest("Could not parse JSON")
+    data = flask.request.json
+    try:
+        _validate_binance_network_update_v1(data)
+    except fastjsonschema.JsonSchemaException:
+        raise exceptions.ValidationException("User input did not match JSON schema")
```

Comment:
ditto

Reply:
(Refer to other examples in rebased code to fix.)

Reply:
Fixed.